docs/UPDATING_DEPENDENCIES.md

```diff
@@ -0,0 +1,120 @@
+# Updating @elastic/synthetics and Playwright Versions
+
+This guide explains how to update the `@elastic/synthetics` package and its associated Playwright dependencies in the synthetics-recorder project.
+
+## Background
+
+This project uses a custom fork of Playwright maintained at [elastic/playwright](https://github.com/elastic/playwright). The fork includes additional patches required for the recorder functionality. When updating `@elastic/synthetics`, you must also update the Playwright packages to matching versions.
+
+## Prerequisites
+
+- Access to the [elastic/playwright](https://github.com/elastic/playwright) repository
+- Access to the [elastic/synthetics](https://github.com/elastic/synthetics) repository
+- Git configured with appropriate permissions to push to elastic/playwright
+
+## Update Process
+
+> **Note:** The examples in this guide illustrate upgrading Playwright from version `1.56.1` to `1.57.0`. Replace these versions with your actual current and target versions.
+
+### Step 1: Identify the Required Playwright Version
+
+Check which Playwright version is used by the target `@elastic/synthetics` version:
+
+1. Go to the [elastic/synthetics](https://github.com/elastic/synthetics) repository
+2. Find the version you want to update to (e.g., `1.22.0`)
+3. Check the `package.json` to identify the `playwright`, `playwright-core`, and `playwright-chromium` versions
+
+### Step 2: Create the New Playwright Branch
+
+In the [elastic/playwright](https://github.com/elastic/playwright) fork:
+
+1. Checkout the corresponding Microsoft Playwright release tag
+
+   ```bash
+   git fetch upstream
+   git checkout v1.57.0  # Replace with target version
+   ```
+
+2. Create a new branch following the naming convention `{version}-recorder`
+   ```bash
+   git checkout -b 1.57.0-recorder
+   ```
+
+### Step 3: Cherry-pick the Recorder Patches
+
+The elastic/playwright fork contains custom commits on top of the Microsoft release. You need to apply these patches to the new branch:
+
+1. Find the current recorder branch being used (check `package.json` in synthetics-recorder). In this example, the current branch is `1.56.1-recorder`.
+2. Identify the extra commits on that branch (commits after the Microsoft `v1.56.1` release tag)
+3. Cherry-pick the **first** patch commit and resolve any conflicts
+
+   ```bash
+   git cherry-pick <first-commit-sha>
+   # Resolve conflicts if any
+   git add .
+   git cherry-pick --continue
+   ```
+
+4. Cherry-pick the **second** patch commit (this typically includes the packaging script)
+   ```bash
+   git cherry-pick <second-commit-sha>
+   ```
+
+### Step 4: Build and Publish the Tarballs
+
+After cherry-picking the commits:
+
+1. Run the packaging script to create the tarballs
```

Comment:
Please include the command to run the script so if someone's following this with no experience they'll be able to succeed without needing to research what is meant here.